ui/mods/msu/keybinds/keybind.js

```diff
@@ -1,43 +1,11 @@
-MSUKeybind = function( _modID, _id, _function, _keyCombinations, _keyState )
+MSUKeybind = function( _modID, _id _keyCombinations )
```

Comment:
What's the distinction between `MSUKeybind` and `MSU.Keybind`? I feel like I've seen both used in a couple places (or maybe it's `MSU.Keybinds`, which. . .I guess I can grok as a collection of keybinds in MSU, vs this, which is a MSU keybind separate from a keybind.nut or keybind_sq.nut. . .?).